frontend/appflowy_flutter/lib/plugins/document/document_page.dart

```diff
@@ -240,7 +242,7 @@ class _AppFlowyEditorPageState extends State<_AppFlowyEditorPage> {
     }
   }
 
-  dartz.Tuple2<bool, Selection?> _autoFocusParameters() {
+  dartz.Tuple2<bool, Selection?> _autoFocusParamters() {
```

Comment:
Why rename, it was correctly spelled before?
```suggestion
  dartz.Tuple2<bool, Selection?> _autoFocusParameters() {
```